tmc-plugin/src/fi/helsinki/cs/tmc/model/NbTmcSettings.java

```diff
@@ -153,9 +156,9 @@ public boolean isSavingPassword() {
 
     @Override
     public String getTmcMainDirectory() {
-        String path = settings.get(PREF_PROJECT_ROOT_DIR, null);
+        Path path = Paths.get(settings.get(PREF_PROJECT_ROOT_DIR, null));
         if (path != null) {
-            return path;
+            return path.toString();
```

Comment:
`Paths.get(null)` causes an NPE. We should guard against that.

Because of the above, the `path` can never be `null` on this line, so we can just get rid of this check.